### PR TITLE
Update kube prometheus

### PR DESCRIPTION
**Pull request by david-martin**
There are 3 related changes in this PR:

- Updating the kube-prometheus version from 0.11 to 0.13. This updates the version of Grafana from 8 to 9.
- As a result of the Grafana update, all example dashboards have be re-exported from Grafana 9
- To make it easier to ensure the dashboards are in the right format for sharing and for configmap use, a make target has been added `make dashboard-cleanup` that runs the already existing `universal-dashboard.sh` script on all dashboard json files. The readme has been udpated to reflect this change.

1 other small change is a height increase to 1 to a table in the app developer dashboard as it is cropped after the upgrade to Grafana 9.

To verify, run the `quickstart-setup.sh` locally, port forward to the grafana service and verify the dashboards all imported OK and look reasonable.

### CODE REVIEW

**Comment by ehearneRedHat** [comment]
:eyes:

File: config/observability/README.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ Some example alerting rules are available in the [/examples](/examples) folder.
 
 ## Exporting a dashboard for use with Grafana Community Platform or other Grafana Instances
 
-Following the steps in [Editing dashboards](#editing-dashboards), we need to make one export where the toggle "Export for sharing manually" is toggled. Once this is saved, run the following from `config/observability` . 
+Following the steps in [Editing dashboards](#editing-dashboards), export the dashboard json with the toggle "Export for sharing manually" on. Once all dashboard json files sare saved/updated, run the following make target to sanitise it for use in both the Grafana Community Platform for sharing, and for use as a mounted configmap volume locally.
```
Comment (ehearneRedHat) [inline config/observability/README.md:52]:
```suggestion
Following the steps in [Editing dashboards](#editing-dashboards), export the dashboard json with the toggle "Export for sharing manually" on. Once all dashboard json files sare saved/updated, run the following make target to sanitise it for use in both the Grafana Community Platform for sharing, and for use as a mounted configmap volume locally.
```
```suggestion
Following the steps in [Editing dashboards](#editing-dashboards), export the dashboard json into `examples/dashboards/` with the toggle "Export for sharing manually" on. Once all dashboard json files sare saved/updated, run the following make target to sanitise it for use in both the Grafana Community Platform for sharing, and for use as a mounted configmap volume locally.
```

**Review by ehearneRedHat — APPROVED** [review]
@david-martin tested locally - all dashboards imported ok and looked reasonable. Also tested the new make target which works well. I made a suggestion to add where to save the files so that the make target can pick them up - feel free to commit suggestion if you like.

Either way, looks good to merge :)

**Comment by david-martin** [comment]
Thanks @ehearneRedHat 
I've added your suggestion.

**Pull request merged**